e2e/logInSignUp.spec.js

```diff
@@ -134,8 +135,22 @@ describe('Log in flow', () => {
 
 describe('testing swiping on feed page', () => {
```

Comment:
Again (For some reason comments won't consistently appear, but we need better naming. Here we are not swiping through the feed but rather through the pages starting with feed!